### PR TITLE
Fix test

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

**Comment by boegel** [comment]
@itkovian: docstrings?

**Comment by JensTimmerman** [comment]
This didn't work, 
try creating your optparser like this: `ExtendedSimpleOption(go_args=[], {})`

**Comment by JensTimmerman** [comment]
or mock `default_parseoptions(self)` to return an empty list

File: test/script_tools.py [hunk 1]
```diff
@@ -42,6 +43,14 @@ class TestExtendedSimpleOption(TestCase):
     """
     Tests for the ExtendedSimpleOption class.
     """
+    def setup(self):
```
Comment (JensTimmerman) [inline test/script_tools.py]: `setUp`

File: test/script_tools.py [hunk 2]
```diff
@@ -42,6 +43,14 @@ class TestExtendedSimpleOption(TestCase):
...
+        """Backup sys.argv"""
+        self._old_argv = sys.argv
+        sys.argv = []
+
+    def teardown(self):
```
Comment (JensTimmerman) [inline test/script_tools.py]: `tearDown`

**Pull request merged**